src/cmds/login.ts

```diff
@@ -44,9 +45,9 @@ export default class LoginCommand extends Command {
   async run(opts: CommandOptions<Options>) {
     super.run(opts);
 
-    let { project } = opts;
+    prompts.override(opts);
```

Comment:
I got irrationally excited about this one lol

Reply:
yeahhh I read about override and login was the first thing I thought of... I don't want to look at the `versions` commands for a long time lmao but I looked into doing this for the `prepareOas` handler and we have enough complexity to our spec searching that I can't think of a useful refactor approach

Reply:
yeah i'd rather remove the versions commands before even thinking about doing anything else to them